exonum-java-binding-core/src/test/java/com/exonum/binding/storage/indices/CheckedMapProofMatcher.java

```diff
@@ -21,74 +21,45 @@
 import com.exonum.binding.common.proofs.map.MapProofStatus;
 import com.exonum.binding.common.serialization.StandardSerializers;
 import com.google.common.io.BaseEncoding;
-import java.util.Arrays;
+import com.google.protobuf.ByteString;
 import java.util.List;
-import java.util.Optional;
+import java.util.Set;
 import java.util.stream.Collectors;
 import org.hamcrest.Description;
+import org.hamcrest.Matcher;
 import org.hamcrest.TypeSafeMatcher;
+import org.hamcrest.core.IsEqual;
 
 class CheckedMapProofMatcher extends TypeSafeMatcher<CheckedMapProof> {
 
   private static final BaseEncoding HEX_ENCODING = BaseEncoding.base16().lowerCase();
 
   private final List<MapTestEntry> entries;
```

Comment:
Maybe we should do it separately, as https://jira.bf.local/browse/ECR-2526 is required beforehand?

Reply:
We can if you document this task properly on Jira

Reply:
Updated ECR-2526 (I don't think it needs a separate task?)
By the way, why do you think that ordered set is needed for `expectedMissingKeys`?

Reply:
_insertion_ ordered set — so that the order in which they appear in the output matches the order in which they were passed to the matcher in the test.